tcms/testruns/templates/testruns/get.html

```diff
@@ -63,39 +63,52 @@ <h2 class="card-pf-title" style="text-align: left">
                     </h2>
                 {% endif %}
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-spinner"></span>{% trans 'Status' %}:
-                    <input class="bootstrap-switch" id="status_button" type="checkbox"
-                        {% if not object.stop_date %}checked{% endif %}
-                        {% if not perms.testruns.change_testrun %}disabled{% endif %}
-                        data-on-text="{% trans 'Running' %}" data-off-text="{% trans 'Finished' %}">
-                </h2>
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned start' %}:
                     {{ object.planned_start|default:'-'}}
                 </h2>
 
-                <h2 class="card-pf-title" style="text-align: left">
-                    <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
-                    {{ object.start_date }}
-                </h2>
+{#                Start date #}
+                {% if object.start_date == None %}
+                    <h2 class="card-pf-title" style="text-align: left">
+                        <span class="fa fa-spinner"></span>{% trans 'Start date' %}:
+                        <input class="bootstrap-switch" id="start_button" type="checkbox"
+
+                               {% if not perms.testruns.change_testrun %}disabled{% endif %}
+                               data-on-text="{% trans 'Started' %}" data-off-text="{% trans 'Planned' %}">
```

Comment:
When start_date is None => the button text should be "Start". The onclick() handler will update the date and also refresh the UI and hide itself. 

When start_date is set then you can't go back so the button will not be visible.


I don't think 2 state buttons make sense here b/c you have only 1 state. A simple button should do the trick, see https://www.patternfly.org/v3/pattern-library/widgets/#buttons + the Save buttons on most forms. Small or Extra Small should fit well into the existing UI. 

For a very minimal button you can use one of the checkmark icons + a title text like Start/Finish, etc: https://fontawesome.com/v4.7.0/icons/

Reply:
should it be something like this or after the click the button should disappear?
![Uploading Screen Shot 2021-04-05 at 11.18.30 AM.png…]()

Reply:
Can't test until the problem with migrations is solved. See https://github.com/kiwitcms/Kiwi/pull/2328#discussion_r606699718